src/client/app/core/styles/index.styl

```diff
@@ -7,14 +7,25 @@ body, html
 
 
 body
-  box-sizing      : border-box
-  display         : flex
-  align-items     : center
-  justify-content : center
-
+  padding:10px
+  background-color:#2C3E50
   *
     box-sizing : border-box
 
+.navbar
```

Comment:
is this being used right now? Eventually we'll use Bootstrap's navbar component and add onto that, so we need to watch out for class name collisions.